common/djangoapps/enrollment/views.py

```diff
@@ -581,8 +586,19 @@ def post(self, request):
                 )
 
             enterprise_course_consent = request.data.get('enterprise_course_consent')
```

Comment:
No, but ecommerce and platform don't get deployed synchronously. So, we need to deploy platform first, and then have it be able to handle both types of requests until the ecommerce changes get deployed.